### PR TITLE
0.13: Update reference docs

### PR DESCRIPTION
**Pull request by Orzelius**

### CODE REVIEW

File: docs/reference/providers/conftest-container.md [hunk 1]
```diff
@@ -7,9 +7,9 @@ tocTitle: "`conftest-container`"
 
 ## Description
 
-This provider automatically generates [conftest modules](../module-types/conftest.md) for `container` modules in your project. A `conftest` module is created for each `container` module that includes a Dockerfile that can be validated.
+This provider automatically generates [conftest Test actions](../action-types/conftest.md) for `container` Builds in your project. A `conftest` Test is created for each `container` Build that includes a Dockerfile that can be validated.
```
Comment (vvagaytsev) [inline docs/reference/providers/conftest-container.md:10]: `../action-types/conftest.md` was reported as a dead link in [check-docs](https://app.circleci.com/pipelines/github/garden-io/garden/17501/workflows/980a01f4-27df-4eda-8e8f-ab8d512907a7/jobs/302702).

File: plugins/conftest-container/index.ts [hunk 2]
```diff
@@ -19,9 +19,9 @@ export const gardenPlugin = () =>
     base: "conftest",
     dependencies: [{ name: "container" }],
     docs: dedent`
-    This provider automatically generates [conftest modules](../module-types/conftest.md) for \`container\` modules in your project. A \`conftest\` module is created for each \`container\` module that includes a Dockerfile that can be validated.
+    This provider automatically generates [conftest Test actions](../action-types/conftest.md) for \`container\` Builds in your project. A \`conftest\` Test is created for each \`container\` Build that includes a Dockerfile that can be validated.
```
Comment (vvagaytsev) [inline plugins/conftest-container/index.ts:22]: [Same](https://github.com/garden-io/garden/pull/4326/files#r1199010557) here.

File: docs/reference/providers/conftest-container.md (hunk 1 above)
Comment (vvagaytsev) [reply docs/reference/providers/conftest-container.md:10]: It should be `../action-types/Test/conftest.md`

File: plugins/terraform/index.ts [hunk 3]
```diff
@@ -54,13 +55,13 @@ const configSchema = providerConfigBaseSchema()
 
         See the [Terraform guide](${docsBaseUrl}/advanced/terraform) for more information.
       `),
-    // When you provide variables directly in \`terraform\` modules, those variables will
+    // When you provide variables directly in \`terraform\` axtions, those variables will
```
Comment (Walther) [inline plugins/terraform/index.ts:58]: typofix
```suggestion
    // When you provide variables directly in \`terraform\` actions, those variables will
```

File: docs/reference/action-types/Deploy/pulumi.md [hunk 4]
```diff
@@ -158,25 +158,6 @@ kind:
...
-          - # POSIX-style path or filename of the directory or file(s) to copy to the target.
-            source:
-
-            # POSIX-style path or filename to copy the directory or file(s), relative to the build directory.
-            # Defaults to the same as source path.
-            target:
-
-    # Maximum time in seconds to wait for build to finish.
-    timeout: 600
-
```
Comment (Walther) [inline docs/reference/action-types/Deploy/pulumi.md:179]: I take it these removals are intentional - no longer inheriting the base build spec for both `pulumi` and `terraform` actions?
(Just to double check I understood correctly)

File: docs/reference/module-types/container.md [hunk 5]
```diff
@@ -244,8 +244,8 @@ services:
 
     # List of volumes that should be mounted when starting the container.
     #
-    # Note: If neither `hostPath` nor `module` is specified, an empty ephemeral volume is created and mounted when
-    # deploying the container.
+    # Note: If neither `hostPath` nor `module` is specified,
```
Comment (Walther) [inline docs/reference/module-types/container.md:247]: Should we keep this `module` terminology in these lines, or do these need to be changed as well?
(repeats below)

File: docs/reference/module-types/jib-container.md [hunk 6]
```diff
@@ -297,8 +297,8 @@ services:
 
     # List of volumes that should be mounted when starting the container.
     #
-    # Note: If neither `hostPath` nor `module` is specified, an empty ephemeral volume is created and mounted when
-    # deploying the container.
+    # Note: If neither `hostPath` nor `module` is specified,
```
Comment (Walther) [inline docs/reference/module-types/jib-container.md:300]: ditto, `module` vs `action` in this file, repeats below in similar notes

File: docs/reference/module-types/container.md (hunk 5 above)
Comment (Orzelius) [reply docs/reference/module-types/container.md:247]: yes we should. This is referring to the [volumes.module field on module config](https://github.com/garden-io/garden/blob/0.13/docs/reference/module-types/container.md#servicesvolumesmodule)

File: docs/reference/module-types/jib-container.md (hunk 6 above)
Comment (Orzelius) [reply docs/reference/module-types/jib-container.md:300]: This is referring to the [volumes.module field on module configs](https://github.com/garden-io/garden/blob/0.13/docs/reference/module-types/container.md#servicesvolumesmodule).

Action configs have action in the same place.

File: docs/reference/module-types/container.md (hunk 5 above)
Comment (Walther) [reply docs/reference/module-types/container.md:247]: gotcha, thanks for the clarification!

**Review by Walther — DISMISSED** [review]
Looks good to me, thanks for all the hard work! ✨

**Comment by Orzelius** [comment]
I ask for a re-approve as I had to rebase the branch

**Review by vvagaytsev — DISMISSED** [review]

**Review by Walther — APPROVED** [review]

**Pull request merged**